setup.py

```diff
@@ -13,7 +13,7 @@
 
 DISTNAME = 'skglm'
 DESCRIPTION = 'A fast and modular scikit-learn replacement for generalized linear models'
-with open('README.rst', 'r') as f:
+with open('README.md', 'r', encoding='utf-8') as f:
```

Comment:
Not adding ``encoding`` will result in reading incorrectly the file. Especially here, I'm using emojis.

cf. https://github.com/mathurinm/celer/pull/277